### ISSUE TITLE
Spark - padding causes an offset in SkinnableContainer (Group is OK)

### ISSUE THREAD

**Issue by yishayw** [apache/royale-asjs#1019]
In this app

```
<?xml version="1.0" encoding="utf-8"?>
<s:Application 
	xmlns:fx="http://ns.adobe.com/mxml/2009"
	xmlns:s="library://ns.apache.org/royale/spark"
	>
	<s:SkinnableContainer id="myParent" style="background-color: green; alpha: 0.5">
		<s:layout>
			<s:VerticalLayout paddingBottom="20" paddingLeft="20" paddingRight="20" paddingTop="20" />
		</s:layout>
		<s:Group id="myContent" height="75" width="588" style="background-color: red; alpha: 0.5"/>
	</s:SkinnableContainer>
</s:Application>

```

The red group (myContent) slides outside its container (myParent). If you make myParent a Group instead of a SkinnableContainer then the problem goes away.

**Comment by yishayw** [comment]
SkinnableContainer's IContentView does get sized correctly, I wonder if in ChildResize branch the SkinnableContainer should detect this and size according to its content view.

**Comment by yishayw** [comment]
Had to revert the fix as it was causing some layout issues.

**Comment by estanglerbm** [comment]
This seems to be fixed, at least with PR #1135 and other previous PRs (don't know which one fixed it).  It shows a green border (20px) and an eye-destroying filled bright red rectangle in the middle, and DOM attributes seem to confirm.
